.circleci/config.yml

```diff
@@ -1,113 +1,156 @@
-  node6_webpack_latest:
+  node10-latest:
     docker:
-      - image: circleci/node:6.13.0-stretch-browsers
+      - image: webpackcontrib/circleci-node10:latest
     <<: *unit_tests
-
+  node8-canary:
+    docker:
+      - image: webpackcontrib/circleci-node8:latest
+    <<: *canary_tests
   analysis:
     docker:
-      - image: circleci/node:9.6.1-browsers
+      - image: webpackcontrib/circleci-node-base:latest
     steps:
       - checkout
-      - setup_remote_docker
       - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
           name: NPM Rebuild
-          command: npm rebuild
+          command: npm install
       - run:
           name: Run linting.
           command: npm run lint
-      # - run:
-      #     name: Run NSP Security Check.
-      #     command: npm run security
+      - run:
+          name: Run NSP Security Check.
+          command: npm run security
+      - run:
+          name: Validate Commit Messages
+          command: npm run ci:lint:commits
+  publish:
+    docker:
+      - image: webpackcontrib/circleci-node-base:latest
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: NPM Rebuild
+          command: npm install
       # - run:
       #     name: Validate Commit Messages
-      #     command: npm run ci:lint:commits
+      #     command: npm run release:validate
+      - run:
+          name: Publish to NPM
+          command: printf "noop running conventional-github-releaser"
 
 version: 2.0
 workflows:
   version: 2
-  run-tests:
+  validate-publish:
     jobs:
       - dependency_cache
-      - node9_webpack_latest:
+      - node6-latest:
           requires:
             - dependency_cache
           filters:
             tags:
               only: /.*/
-      - node8_webpack_latest:
+      - analysis:
           requires:
             - dependency_cache
           filters:
             tags:
               only: /.*/
-      - node6_webpack_latest:
+      - node8-latest:
           requires:
-            - dependency_cache
+            - analysis
+            - node6-latest
           filters:
             tags:
               only: /.*/
-      - analysis:
+      - node10-latest:
+          requires:
+            - analysis
+            - node6-latest
+          filters:
+            tags:
+              only: /.*/
+      - node8-canary:
           requires:
-            - node9_webpack_latest
-            - node8_webpack_latest
-            - node6_webpack_latest
+            - analysis
+            - node6-latest
           filters:
             tags:
               only: /.*/
+      - publish:
+          requires:
+            - node8-latest
+            - node8-canary
+            - node10-latest
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
_nit:_ Please add a blank line at the end of the file

Reply:
(sorry, not repeating to be a silly person) webpack-defaults again oyyyyeeee